Block/Slices.php

```diff
@@ -6,7 +6,7 @@
 
 class Slices extends Group
 {
-    public function fetchItem(\stdClass $slice): string
+    public function fetchItem(\stdClass $slice, $key): string
```

Comment:
:*suggestion* (non-blocking): you could add the `setIterator($key)` here too on the block after `setDocument`